### PR TITLE
LEED Summary EAp2-4/5 report row shown in uppercase and document special end-use subcategory tags

### PR DESCRIPTION
**Pull request by JasonGlazer**
Addresses #5364 
- [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
- [X] Code style (parentheses padding, variable names)
- [x] Functional code review (it has to work!)
- [x] If defect, results of running current develop vs this branch should exhibit the fix
- [x] CI status: all green or justified
- [X] Performance: CI Linux results include performance check -- verify this
- [X] Unit Test(s)

### CODE REVIEW

**Comment by JasonGlazer** [comment]
Reviewed the diffs and they all look correct. The AUD files are showing an increase of one for the numTableEntry which makes sense if the comparison is sensitive to capitalization now. The differences in the five files with big Table diffs also seem reasonable. They are cases where the original tables were showing all caps in the object count summary because the name of the objects were matching the type of object. With the fix, they are shown in normal title case like we would prefer.

**Comment by JasonGlazer** [comment]
Looks like this is ready to go. The same AUD differences and and Table diffs as I reviewed yesterday and looked fine.

**Comment by nrel-bot-3** [comment]
@JasonGlazer @lgentile it has been 14 days since this pull request was last updated.

File: doc/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex [hunk 1]
```diff
@@ -72,7 +72,10 @@ \subsection{Baseline Building Rotations}\label{baseline-building-rotations}
...
+
+The EAp2-4/5 Performance Ration Method Compliance subtable has some special options in order to show end uses that are different than the end-uses that are typically used with EnergyPlus. By specifing the end-use subcategories of "Cooking" most commonly with the ElectricEquipment and GasEquipment objects, "Elevators and Escalators" most commonly with the ElectricEquipment object, "Industrial Process" most commonly with the ElectricEquipment and GasEquipment objects, "Interior Lighting - Process" most commonly with the Lights object, or "Fans - Parking Garage" most commonly with the Exterior:FuelEquipment object, those end uses are shown in the EAp2-4/5 Performance Ration Method Compliance subtable.
```
Comment (mjwitte) [inline doc/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex]: "Ration" --> "Rating" (happens twice in this sentence.

File: src/EnergyPlus/OutputReportTabular.cc [hunk 2]
```diff
@@ -10210,8 +10211,8 @@ namespace OutputReportTabular {
 			curObjectName = tableEntry( lTableEntry ).objectName;
 			found = 0;
 			for ( mUnqObjNames = 1; mUnqObjNames <= numUnqObjName; ++mUnqObjNames ) {
-				if ( SameString( curObjectName, uniqueObjectName( mUnqObjNames ) ) ) {
-					found = mUnqObjNames;
+				if ( curObjectName == uniqueObjectName( mUnqObjNames ) ) {
+						found = mUnqObjNames;
```
Comment (mjwitte) [inline src/EnergyPlus/OutputReportTabular.cc]: Looks like this is indented one too far - but may just be the github display.

**Comment by mjwitte** [comment]
- [x] 1. In the table, it seems like "Interior Lighting (process)" should be "Interior Lighting-Process" to match up with "Fans-Interior" and "Fans-Parking Garage".
- [x] 2. For these end-uses, the table has no spaces around the dash, but the strings expected in the end-use subcategory field do have spaces.  Seems like no spaces would be better, but we could accept them both.
- [x] 3. We should add at least one object for each of these to an existing example file.
- [x] 4. For each object that accepts these, it would be good to mention the applicable sub-enduse strings in the IDD \note

**Comment by JasonGlazer** [comment]
@mjwitte Concerning your item 4, lots of objects take contain an End-Use Subcategory field and any of them could be used with these special subcategory tags. The objects include:
- Lights*
- ElectricEquipment*
- GasEquipment*
- HotWaterEquipment*
- SteamEquipment*
- ElectricEquipment:ITE:AirCooled
- ZoneBaseboard:OutdoorTemperatureControlled
- Exterior:Lights*
- Exterior:FuelEquipment*
- Exterior:WaterEquipment*
- Fan:ConstantVolume
- Fan:VariableVolume
- Fan:OnOff
- Fan:ZoneExhaust
- Fan:ComponentModel
- WaterHeater:Stratified
- EnergyManagementSystem:MeteredOutputVariable
- Refrigeration:CompressorRack
- Refrigeration:Condenser:AirCooled
- Refrigeration:Condenser:EvaporativeCooled
- Refrigeration:Condenser:WaterCooled
- Refrigeration:GasCooler:AirCooled
- Refrigeration:Compressor
- Refrigeration:System
- Refrigeration:TranscriticalSystem
- Refrigeration:SecondarySystem
- WaterUse:Equipment

But I think only the ones shown with asterisks make sense to add the note, do you agree?

**Comment by JasonGlazer** [comment]
@mjwitte, how about the following text added to each of those objects End-Use Subcategory field note:

```
   \note Any text may be used here to categorize the end-uses and they are shown in the ABPUPS 
   \note End Uses by Subcategory table but special tags will also place them in certain rows 
   \note in the LEED Summary table  EAp2-4/5. Performance Rating Method Compliance. The special are
   \note Fans-Parking Garage, Interior Lighting-Process, Cooking, Industrial Process, Elevators and Escalators
```

I should probably add it to the IOref also. What do you think?

**Comment by mjwitte** [comment]
@JasonGlazer The text sounds good with a few minor edits.  I agree with the starred objects, and adding to the I/O Ref.

```
   \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
   \note The following special tags will also place the end-use in specific rows in the LEED Summary table
   \note EAp2-4/5. Performance Rating Method Compliance:  
   \note Fans-Parking Garage, Interior Lighting-Process, Cooking, Industrial Process, Elevators and Escalators.
```

File: doc/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex (hunk 1 above)
Comment (mjwitte) [inline doc/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex]: Noticed a stray tilde, or is that suppsed to be there?  "The~ LEED Summary report provides . . . "

**Comment by JasonGlazer** [comment]
@mjwitte I reviewed many of the diffs and they are as expected. Note that the the 5ZoneEndUses.idf file was changed to add new equipment which resulted in cascading changes. This is ready for your further review.

**Comment by mjwitte** [comment]
@JasonGlazer Tested this with the defect file and with 5ZoneEndUses before and after.  All looks good, but puzzled that the CI table diffs for [5ZoneEndUses](http://energyplus.s3-website-us-east-1.amazonaws.com/regressions/2016-04/829a34d2e8ff09006accdb663a654cedf7633f9b-76febf191778f87b0000b61b73c9d94065bb205d/5ZoneEndUses/x86_64-Linux-Ubuntu-14.04-gcc-4.8/) shows no diffs for the LEED table that is clearly changing.  Other tables show diffs, but not FullName:LEED Summary_Entire Facility_EAp2-4/5. Performance Rating Method Compliance.

**Comment by mjwitte** [comment]
Realized now why some of the table diffs don't report here - because these are only design day runs, so some of the tables are all zeros.  Satisfied that this is working correctly.  Merging.

**Comment by xwanguk** [comment]
V8.6.0 is still with the problem of input for sub-category "Interior lighting - process" as it will offset "Interior Lighting" reported in EAp2-4/5 Performance Rating Method Compliance. The extreme case would be the "Interior Lighting" is sometimes in negative!
Also, it looks like the receptacle is still double counted. Should "Receptacle" be a separate sub-category?

**Comment by JasonGlazer** [comment]
@xwanguk could you please email me an IDF file that shows the problem you are describing. 
jglazer at gard dot com

**Comment by xwanguk** [comment]
I cannot upload here, it says it does support this format but actually I have used a zip which is in their supported list.

**Comment by JasonGlazer** [comment]
Please email me the file. Use jglazer at gard dot com but substitute the at and dot for the normal symbols.

**Comment by xwanguk** [comment]
Sorry, just notice you have given your email address in your previous reply. Will send directly.

**Comment by JasonGlazer** [comment]
@xwanguk thank you for reporting this problem. I added a new issue #5895

**Pull request merged**